signatures/Picnic/Picnic-UR/param/L1-UR.yaml

```diff
@@ -0,0 +1,13 @@
+name: L1-UR
+security level: 
+  classical:  128
+  quantum:  64
+  nist category: 1
+failure probability: 0
+number of operations: inf
+sizes:
+  sk: 16
+  pk: 32
+  ct|sig: 53961
+  msg: 256
+  comment: The size of signature is the maximal possible size.
```

Comment:
If I understand the paper correctly, the sig size does not vary for UR, so we can remove this comment, right?